### ISSUE TITLE
Implement Support Vector Regression (SVR) for Predicting Target Variable

### ISSUE THREAD

**Issue by Mubashir-Ul-Hassan** [Mubashir-Ul-Hassan/data-science-projects#28]
We need to integrate a Support Vector Regression (SVR) model into the project for predicting the target variable using the given dataset. The proposed model will use a linear kernel with the parameters set as C=0.1 and gamma=1.

Tasks:
Add the following SVR implementation:

```
model = svm.SVR(kernel='linear', C=0.1, gamma=1)
model.fit(x_train, y_train)

y_pred = model.predict(x_test)
print(accuracy_score(y_test, y_pred))
```

Train the model using the provided training dataset (x_train, y_train).
Predict outcomes for the test dataset (x_test).
Evaluate the model's performance using an appropriate accuracy metric (SVR typically uses R² score, so consider changing accuracy_score to r2_score).
Expected Outcome:
Implement the SVR model correctly and ensure the prediction results are in line with the dataset’s context.
Modify and replace accuracy_score with an appropriate regression metric like r2_score.

**Comment by VadisettyRahul** [comment]
I'm interested @Mubashir-Ul-Hassan  please assign to me.

**Comment by Mubashir-Ul-Hassan** [comment]
@VadisettyRahul assign to you.
